stacks-signer/CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ and this project adheres to the versioning scheme outlined in the [README.md](RE
 
 ### Changed
 
+- Added tenure extend timestamp to signer block responses
+- Added tenure_idle_timeout_secs configuration option for determining when a tenure extend will be accepted
```

Comment:
Such a nit: but should prob say "when a time-based tenure extend will be accepted" since there are other reasons we might issue one.

Reply:
👍 3bccb233c